src/screens/CreatePost/index.js

```diff
@@ -825,7 +828,10 @@ const CreatePost = () => {
             </>
           )}
           <Gap style={styles.height(25)} />
-          <Button disabled={isButtonDisabled()} onPress={postV2}>
+          <Button
+            disabled={isButtonDisabled()}
+            onPress={postV2}
+            style={styles.buttonPost(useDynamicColors(typeUser))}>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The dynamic color usage here is a good approach for theme handling. However, make sure that the `useDynamicColors` hook returns an object with a `primary` property.
```diff
-            style={styles.buttonPost(useDynamicColors(typeUser))}>
+            style={styles.buttonPost(useDynamicColors(typeUser).primary)}>
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->